SwiftWisdomTests/SwiftWisdomTests.swift

```diff
@@ -41,3 +41,23 @@ class GetConstraintTests: XCTestCase {
         XCTAssertNil(constraint, "Constraint found, expected nil constraint")
     }
 }
+
+class NavigationItemBackButtonTests: XCTestCase {
+
+    func testHidingNavigationBackTitle() {
+        let firstViewController = UIViewController()
+        firstViewController.title = "First"
+        let secondViewController = UIViewController()
+        secondViewController.title = "Second"
+        let thirdViewController = UIViewController()
+        thirdViewController.title = "Third"
+
+        let navigationController = UINavigationController(rootViewController: firstViewController)
+        navigationController.pushViewController(secondViewController, animated: true)
+        XCTAssertNil(firstViewController.navigationItem.backBarButtonItem)
+        secondViewController.ip_hideBackButtonTitle()
+        navigationController.pushViewController(thirdViewController, animated: true)
```

Comment:
This test doesn't actually care whether or not you push a view controller since you're not checking the screen or properties on the navigation controller.

Is there a good way for you to check the property of the back button?